tests/test_storage.py

```diff
@@ -2,36 +2,293 @@
 # -*- coding: utf-8 -*-
 
 import pytest
-from exoscale.api.storage import *
 from .conftest import _random_str
+from datetime import datetime
+from exoscale.api import ResourceNotFoundError
+from exoscale.api.storage import *
```

Comment:
[nitpick] Import star is generally seen as a bad practice ;) (Same remark goes for most of the other test files).

Reply:
I know and agree with this principle, however in this case I'm ok to bend the rules a little since we're importing our own stuff, there's little risk of namespace collision here 😉